### DISCUSSION TITLE
Is there a way to install Lutris and POL Programs directly instead of importing?

### DISCUSSION THREAD

**Discussion by ghost** [bottlesdevs/Bottles#197] [Q&A] [answered]
So that I can just run some Lutris and POL installers and then run those programs instead of needing to import them after installing.

**Comment by mirkobrombin** [comment, marked as answer]
No. But.. Bottles v4 will come with Installers support, these are installation scripts similar to those offered by Lutris and POL, so you can convert them to ease the installation process.

Keep in mind that Lutris is a step further on automating installations and I recommend you take advantage of that (at least for now).
